includes/class-job-dashboard-shortcode.php

```diff
@@ -51,8 +53,13 @@ public function __construct() {
 
 		add_filter( 'paginate_links', [ $this, 'filter_paginate_links' ], 10, 1 );
 
-		add_action( 'job_manager_job_dashboard_column_date', [ $this, 'job_dashboard_date_column_expires' ] );
-		add_action( 'job_manager_job_dashboard_column_job_title', [ $this, 'job_dashboard_title_column_status' ] );
+		add_action( 'job_manager_job_dashboard_column_date', [ self::class, 'the_date' ] );
```

Comment:
I feel that this is a change that might be worth covering in the changelog.

Otherwise, users that have customized the template might find weird that the date and job title are now appearing duplicated.

WDYT?

Reply:
Will do, together with the template change. These actions are new and were just added in a previous PR — previously actions were [only ran for custom columns](https://github.com/Automattic/WP-Job-Manager/blob/trunk/templates/job-dashboard.php#L85). Now it's a bit more flexible to change the data with hooks or add to multiple things to a column, related to compressing this table to work better on mobile.